### PR TITLE
feat: add gen api @doc comment to logic handler routes

### PR DESCRIPTION
**Pull request by ch3nnn**
example:

demo.api
```
syntax = "v1"

type Request {
	Name string `path:"name,options=you|me"`
}

type Response {
	Message string `json:"message"`
}

service demo-api {

	@doc "demo comment"
	@handler DemoHandler
	get /from/:name(Request) returns (Response)

	@doc ("demo list comment")
	@handler DemoListHandler
	get /from/list returns (Response)

	@doc(
		summary: "comment comment comment"
	)
	@handler NoResponseHandler
	get /from/get (Request)
}
```
generate  logic  handler routes comment

image

image

image

### CODE REVIEW

**Comment by ch3nnn** [comment]
@kevwan I hope you can help me see if this change is reasonable and correct. thank!

File: tools/goctl/api/gogen/handler.tpl [hunk 1]
```diff
@@ -1,26 +1,27 @@
-package {{.PkgName}}
```
Comment (kesonan) [inline tools/goctl/api/gogen/handler.tpl:1]: Do not modify template variables to prevent incompatibility problems for users who customize templates.

Comment (ch3nnn) [reply tools/goctl/api/gogen/handler.tpl]: ok,  new record has been submitted.  @kesonan

**Review by kesonan — APPROVED** [review]
LGTM

**Comment by ch3nnn** [comment]
i see change approved, want to know the current PR can be merged ? @kevwan

**Pull request merged**